pkg/api/handlers/compat/images.go

```diff
@@ -398,3 +398,45 @@ func LoadImages(w http.ResponseWriter, r *http.Request) {
 		Stream: fmt.Sprintf("Loaded image: %s\n", id),
 	})
 }
+
+func ExportImages(w http.ResponseWriter, r *http.Request) {
+	// 200 OK
+	// 500 Error
+	decoder := r.Context().Value("decoder").(*schema.Decoder)
+	runtime := r.Context().Value("runtime").(*libpod.Runtime)
+
+	query := struct {
+		Names string `schema:"names"`
+	}{
+		// This is where you can override the golang default value for one of fields
+	}
+	if err := decoder.Decode(&query, r.URL.Query()); err != nil {
+		utils.Error(w, "Something went wrong.", http.StatusBadRequest, errors.Wrapf(err, "failed to parse parameters for %s", r.URL.String()))
+		return
+	}
+	var images []string
+	for _, name := range strings.Split(query.Names, ",") {
+		images = append(images, name)
+	}
+	tmpfile, err := ioutil.TempFile("", "api.tar")
+	if err != nil {
+		utils.Error(w, "Something went wrong.", http.StatusInternalServerError, errors.Wrap(err, "unable to create tempfile"))
+		return
+	}
+	if err := tmpfile.Close(); err != nil {
+		utils.Error(w, "Something went wrong.", http.StatusInternalServerError, errors.Wrap(err, "unable to close tempfile"))
+		return
+	}
+	if err := runtime.ImageRuntime().SaveImages(r.Context(), images, "docker-archive", tmpfile.Name(), false); err != nil {
```

Comment:
I do not see where in SaveImages() the `repositories` file is created.

Reply:
i personally see that as out of scope.  i would like to treat it as a bug when "we" hit it. cool with that?